<launch>
    <node name="dino" pkg="dino" type="dino_node" output = "screen">
    	<param name = "service_name" 	type = "str" value = "make_a_dino"/>
    	<param name = "topic_name" 	    type = "str" value = "dinosaurs"/>

    </node>
</launch>